app/src/main/res/layout/activity_git_log.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright © 2014-2020 The Android Password Store Authors. All Rights Reserved.
+  ~ SPDX-License-Identifier: GPL-3.0-only
+  -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/git_log_recycler_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:scrollbars="vertical"
+        tools:context="com.zeapo.pwdstore.git.log.GitLogActivity"
+        tools:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
```

Comment:
```suggestion
        app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
```

This will set the layout manager directly, making it work for both the preview as well as the actual app.